src/main/resources/blogs/bloggers.json

```diff
@@ -1001,6 +1001,12 @@
       "name": "Bartłomiej Szwej",
       "rss": "https://medium.com/feed/@bszwej",
       "twitter": "@bszwej"
+    },
+    {
+      "bookmarkableId": "dmydlarz",
```

Comment:
Any reason why you prefer `dmydlarz` over 'dariusz-mydlarz` as used in other entries?